### PR TITLE
[BACKLOG-35956] : Fix Search Bar

### PR DESCRIPTION
**Pull request by addagudi**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/excel/impl,plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java [hunk 1]
```diff
@@ -56,13 +56,8 @@
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.custom.CLabel;
 import org.eclipse.swt.custom.SashForm;
-import org.eclipse.swt.events.MouseAdapter;
-import org.eclipse.swt.events.MouseEvent;
+import org.eclipse.swt.events.*;
```
Comment (peterrinehart) [inline plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java:59]: Please change this back to individual import statements.  We don't want to import the whole package if we don't need it.

**Review by peterrinehart — CHANGES_REQUESTED** [review]

File: plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java (hunk 1 above)
Comment (addagudi) [reply plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java]: Done

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/excel/impl,plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
plugins/excel/impl,plugins/file-open-save-new/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by peterrinehart — APPROVED** [review]

**Comment by addagudi** [comment]
Will create a new one

**Pull request closed without merging**